### PR TITLE
MAUI 874348 - Added known issues in DataForm and ImageEditor controls

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
### Description
Added known issues in DataForm and ImageEditor controls.
Content review task : https://dev.azure.com/Syncfusion-General/Documentation/_workitems/edit/765187

### CODE REVIEW

File: MAUI/DataForm/floating-label-layout.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ Dataform supports the floating label layout for the following editors.
 * Date picker
 * Time picker
 
+N> The time picker will not work in windows platform within text input layout.
```
Comment (florenceliliana) [inline MAUI/DataForm/floating-label-layout.md:86]: N> The time picker will not work in Windows platform within the text input layout.

File: MAUI/DataForm/floating-label-layout.md [hunk 2]
```diff
@@ -868,4 +870,6 @@ N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-dataform/t
 ## Unsupported editors
 The Floating label layout does not support non-editable editors such as [RadioGroup editor](https://help.syncfusion.com/maui/dataform/editors#radiogroup-editor), [CheckBox editor](https://help.syncfusion.com/maui/dataform/editors#checkbox-editor), and [Switch editor](https://help.syncfusion.com/maui/dataform/editors#switch-editor).
 
-N> By default, the layout type of unsupported editors is `Default`. You can also set the layout type for the unsupported editor using this [Changing layout type of the DataFormItem](https://help.syncfusion.com/maui/dataform/floating-label-layout#changing-the-layout-type-for-the-editor).
\ No newline at end of file
+N> By default, the layout type of unsupported editors is `Default`. You can also set the layout type for the unsupported editor using this [Changing layout type of the DataFormItem](https://help.syncfusion.com/maui/dataform/floating-label-layout#changing-the-layout-type-for-the-editor).
+
+N> The time picker will not work in windows platform within text input layout.
```
Comment (florenceliliana) [inline MAUI/DataForm/floating-label-layout.md:875]: N> The time picker will not work in the Windows platform within the text input layout.

File: MAUI/DataForm/migration.md [hunk 3]
```diff
@@ -1371,4 +1371,8 @@ The common properties of `Xamarin SfDataForm` list items ( [ DataFormPickerEdito
 * [MacCatalyst, iOS] - In RTL FlowDirection, the ScrollView scroll bar shows at the right.
 * [Android] - in RTL mode Radio button's content does not arrange from the right.
 * `RTL` flow direction not working for ComboBox and AutoComplete editors.
-* `IsReadOnly` not working for AutoComplete editor.
\ No newline at end of file
+* `IsReadOnly` not working for AutoComplete editor.
+* Dark theme and light theme - There is no border and focused border stroke customization for the default editors (Text, password, multiline text, picker, DatePicker and TimePicker)
```
Comment (florenceliliana) [inline MAUI/DataForm/migration.md:1375]: * Dark and light theme - There is no border and focused border stroke customization for the default editors (Text, password, multiline text, picker, DatePicker, and TimePicker).

File: MAUI/DataForm/migration.md [hunk 4]
```diff
@@ -1371,4 +1371,8 @@ The common properties of `Xamarin SfDataForm` list items ( [ DataFormPickerEdito
...
+* Dark theme and light theme - [iOS] There is no border and focused border stroke customization for the default editor (Text, password, picker, DatePicker and TimePicker)
```
Comment (florenceliliana) [inline MAUI/DataForm/migration.md:1376]: * Dark and light theme - [iOS] There is no border and focused border stroke customization for the default editor (Text, password, picker, DatePicker, and TimePicker).

File: MAUI/DataForm/migration.md [hunk 5]
```diff
@@ -1371,4 +1371,8 @@ The common properties of `Xamarin SfDataForm` list items ( [ DataFormPickerEdito
...
+* Dark theme and light theme -[MacOS] There is no border and focused border stroke customization for the default editor (Text, password and picker)
+* Dark theme and light theme - MacOS The text color customization will not work for the DatePicker and TimePicker.
```
Comment (florenceliliana) [inline MAUI/DataForm/migration.md:1378]: * Dark and light theme - MacOS The text color customization will not work for the DatePicker and TimePicker.

File: MAUI/DataForm/floating-label-layout.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/DataForm/floating-label-layout.md]: Updated

File: MAUI/DataForm/migration.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/DataForm/migration.md]: Updated

File: MAUI/DataForm/migration.md [hunk 6]
```diff
@@ -1371,4 +1371,8 @@ The common properties of `Xamarin SfDataForm` list items ( [ DataFormPickerEdito
 * [MacCatalyst, iOS] - In RTL FlowDirection, the ScrollView scroll bar shows at the right.
 * [Android] - in RTL mode Radio button's content does not arrange from the right.
 * `RTL` flow direction not working for ComboBox and AutoComplete editors.
-* `IsReadOnly` not working for AutoComplete editor.
\ No newline at end of file
+* `IsReadOnly` not working for AutoComplete editor.
+* Dark theme and light theme - There is no border and focused border stroke customization for the default editors (Text, password, multiline text, picker, DatePicker and TimePicker)
+* Dark theme and light theme - [iOS] There is no border and focused border stroke customization for the default editor (Text, password, picker, DatePicker and TimePicker)
+* Dark theme and light theme -[MacOS] There is no border and focused border stroke customization for the default editor (Text, password and picker)
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/DataForm/migration.md]: Updated

File: MAUI/DataForm/migration.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply MAUI/DataForm/migration.md]: Updated

**Pull request merged**